docs/animations/Light.md

```diff
@@ -8,7 +8,10 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, light, light anim
 
 # Light
 
-The Light animation behavior performs a point light (A point source of light that emits light in all directions) in the middle of a given UIElement. You set the distance property of the light to determine how bright the light will be. The closer the light source, the darker the UI element will be. ***NOTE**:  Heavy usage of effects may have a negative impact on the performance of your application.* 
+The [Light animation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions) behavior performs a point light (A point source of light that emits light in all directions) in the middle of a given UIElement. You set the distance property of the light to determine how bright the light will be. The closer the light source, the darker the UI element will be. 
```

Comment:
Possible wrong link or intended to use with id #Microsoft_Toolkit_Uwp_UI_Animations_AnimationExtensions_Light_